crates/edr_provider/src/data.rs

```diff
@@ -763,7 +830,8 @@ mod tests {
                 vec![impersonated_account],
             );
 
-            let mut provider_data = ProviderData::new(runtime, &config).await?;
+            let runtime = runtime::Handle::try_current()?;
+            let mut provider_data = ProviderData::new(&runtime, &config).await?;
```

Comment:
Changed it so that we pass in the current runtime handle here instead of on initialization to avoid repetition, since this fixture is only used here.